### PR TITLE
Automate tiefling feats

### PR DESCRIPTION
**Pull request by Shandyan**
Plus a missed tengu feat, an aasimar feat, adding the missed strikes onto a conrasu feat (oops), and adding icons to an automaton strike feat.

### CODE REVIEW

File: packs/data/feats.db/form-of-the-fiend.json [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+                        "value": "jaws"
+                    },
+                    {
+                        "label": "PF2E.BattleForm.Attack.Tail",
+                        "value": "tail"
+                    }
+                ],
+                "key": "ChoiceSet",
+                "prompt": "PF2E.SpecificRule.Tiefling.FormOfTheFiend.Prompt",
+                "rollOption": "formfiend"
```
Comment (TikaelSol) [inline packs/data/feats.db/form-of-the-fiend.json:49]: Should probably make this `form-fiend`.

File: static/lang/re-en.json [hunk 2]
```diff
@@ -1530,6 +1530,11 @@
                     "Prompt": "Select an element."
                 }
             },
+            "Tengu": {
+                "TenguWeaponFamiliarity": {
+                    "Prompt": "Select a weapon."
```
Comment (TikaelSol) [inline static/lang/re-en.json:1535]: Do we really not have these prompts elsewhere? I think there's a general prompt section.

Comment (Shandyan) [reply static/lang/re-en.json:1535]: I can only find 'Select a weapon group' in re-en.json, which isn't right for this feat

File: packs/data/feats.db/form-of-the-fiend.json (hunk 1 above)
Comment (Shandyan) [reply packs/data/feats.db/form-of-the-fiend.json]: Oh yes. Done

File: static/lang/re-en.json (hunk 2 above)
Comment (TikaelSol) [reply static/lang/re-en.json:1535]: Let's move them under SpecificRule.Prompt so everything can use them.

Comment (Shandyan) [reply static/lang/re-en.json]: Looking again this morning, there's a specific weapon prompt right above the weapon group one. So the tengu feat now points to that, and I've deleted the tengu weapon prompt options.

File: static/lang/re-en.json [hunk 3]
```diff
@@ -1544,6 +1544,11 @@
                 },
                 "TalismanEsoterica": "Talisman Esoterica"
             },
+            "Tiefling": {
+                "FormOfTheFiend": {
+                    "Prompt": "Select an unarmed attack."
```
Comment (TikaelSol) [inline static/lang/re-en.json:1549]: I think this one can move to the generic section too.

**Pull request merged**